nservicebus/rabbitmq/routing-topology_support-delayed-delivery_rabbit_[4,].partial.md

```diff
@@ -0,0 +1,7 @@
+### Delayed delivery
+
+Starting with Version 4.3, it is possible to support [delayed delivery](delayed-delivery.md) without requiring the [timeout manager](/nservicebus/messaging/timeout-manager.md) by implementing `ISupportDelayedDelivery` in addition to `IRoutingTopology` in the routing topology class.
```

Comment:
Looking at the initial sentence, perhaps it would be better as:

In Versions 4.3 and above, custom routing topologies can support delayed delivery..."